pkgs/development/python-modules/sslyze/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, lib, callPackage, fetchPypi, fetchFromGitHub, buildPythonPackage, nassl, cryptography, cryptography_vectors, typing-extensions, faker }:
+let
+  cryptographyVersion = "2.9";
+  cryptography_vectors29 = cryptography_vectors.overridePythonAttrs (o: rec {
+    version = cryptographyVersion;
+    src = fetchPypi {
+      inherit version;
+      pname = o.pname;
+      sha256 = "1h7dcgwrjxqk1bzkangbvlhhlgyqd7cfi894dv1cd5m2sp7csblc";
+    };
+  });
+  cryptography29 = (cryptography.override {
+    cryptography_vectors = cryptography_vectors29;
+  }).overridePythonAttrs (o: rec {
+    version = cryptographyVersion;
+    src = fetchPypi {
+      inherit version;
+      pname = o.pname;
+      sha256 = "0vlqy2pki0fh1h6l6cbb43z3g2n9fv0849dzb5gqwjv0bkpx7b0c";
+    };
+  });
+in
```

Comment:
unfortunately this isn't allowed in `python-modules` as it can create an incoherent python package set when paired with other python packages. Please try first to relax the version bounds.

Reply:
Okay, I wasn't aware of that but it seems that 2.9.1 works as well.

Reply:
Done!